src/wallet/wallet.cpp

```diff
@@ -6746,11 +6728,7 @@ bool CWallet::ParameterInteraction(const CChainParams& params)
 {
     if (mapArgs.count("-mintxfee"))
     {
-        CAmount n = 0;
-        if (ParseMoney(mapArgs["-mintxfee"], n) && n > 0)
-            CWallet::minTxFee = CFeeRate(n);
-        else
-            return UIError(AmountErrMsg("mintxfee", mapArgs["-mintxfee"]));
+        UIWarning(_("The argument -mintxfee is no longer supported."));
     }
```

Comment:
I think the check on `maxTxFee` at lines 6755-6759 no longer makes sense. I suggest requiring `maxTxFee` to be at least `conventional_fee(10)` to allow 10-action transactions at the ZIP 317 fee.